BookPlayer/Models/Playlist+CoreDataClass.swift

```diff
@@ -100,27 +115,43 @@ public class Playlist: LibraryItem {
         guard let index = self.itemIndex(with: url) else {
             return nil
         }
+
         return self.getBook(at: index)
     }
 
     func getBook(with identifier: String) -> Book? {
         guard let index = self.itemIndex(with: identifier) else {
             return nil
         }
+
         return self.getBook(at: index)
     }
 
+    override func getBookToPlay() -> Book? {
+        guard let books = self.books else { return nil }
+
+        for item in books {
+            guard let book = item as? Book, !book.isCompleted else { continue }
+
+            return book
+        }
+
+        return nil
+    }
+
+    func getNextBook(after book: Book) -> Book? {
+        guard let books = self.books?.array as? [Book] else { return nil }
```

Comment:
```suggestion
        guard 
          let books = self.books?.array as? [Book],
          let index = books.firstIndex(of: book),
          index + 1 <= books.count else { 
            return nil 
        }
        
        return books[index + 1]
```
Trying out the new Github feature, although it doesn't have an option to let me affect more lines :/

Reply:
I'm not sure if I still like squeezing everything in a single guard just because it's possible. I did before, but I think it gets very hard to read. Maybe this requires further discussion elsewhere.

Reply:
I see your point about readability, and in some cases it may make sense to split the guards. The root of my suggestion is changing the `if let` into a `guard`, to adhere to the [golden path](https://github.com/raywenderlich/swift-style-guide#golden-path), once I saw:
```swift
guard let books = self.books?.array as? [Book] else { return nil }

guard let index = books.firstIndex(of: book), index + 1 <= books.count else { return nil }

return books[index + 1]
```
my mind grouped the first two guards